db/seeds/development.rb

```diff
@@ -0,0 +1,17 @@
+
+n = rand(1..2000)
+12.times do
```

Comment:
`12.times do |n|` lets you use `n` rather than doing it the way you're doing it now on lines 2 and 16.